imports/plugins/core/orders/client/containers/invoiceContainer.js

```diff
@@ -224,8 +224,7 @@ class InvoiceContainer extends Component {
     const paymentMethodName = orderBillingInfo.paymentMethod && orderBillingInfo.paymentMethod.paymentSettingsKey;
     const paymentMethod = Packages.findOne({ _id: paymentMethodId });
     const isRefundable = paymentMethod && paymentMethod.settings && paymentMethod.settings[paymentMethodName]
-      && paymentMethod.settings[paymentMethodName].support.includes("Refund");
-
+      && paymentMethod.settings[paymentMethodName].support.refund;
```

Comment:
I think we need to move this check to a method or something so we can give a safer check. This makes a lot of assumptions about the entire object being there. Or maybe just use `_.get` or equivalent